src/score_components/find-problematic-comments.js

```diff
@@ -10,7 +10,10 @@ module.exports = function grepForProblematicComments(ext, include, exclude) {
   } else {
     find += ' .';
   }
-  find += ` -name "*.${ext}"`;
+
+  const conditions = ext.map((ex) => `-name "*.${ex}"`).join(' -o ');
```

Comment:
Nice! We should update the README to explain the change to how the exclude input is now handled (i.e. if multiple are provided, they are logically OR'ed together)

Reply:
Exclude is the same as before just adding up lots of --not-path into the find command and if no excludes specified, check the gitignore.
Extensions are logically OR'ed.
Updated readme.